### DISCUSSION TITLE
Bouncy notifications!

### DISCUSSION THREAD

**Discussion by BlackDiamond3180** [ppy/osu#21834] [Ideas] [answered]
Having the flick to hide notifications is very cool and all, but how about adding something that makes it bounce across the sides of your screen if you flick it hard enough, maybe even squish or jiggle a little bit when it does so?

**Comment by peppy** [comment, marked as answer]
too much

**Comment by camellirite** [comment]
try flicking it to the left if you don't need it. it's pretty fun!
